README.md

```diff
@@ -82,7 +82,7 @@ pak::pkg_install("cmu-delphi/epidatr@dev")
 ```
 
 Our CRAN listing is
-[here](https://CRAN.R-project.org/package=epidatr/index.html).
+[here](https://cran.r-project.org/web/packages/epidatr/index.html).
```

Comment:
The previous version of this file is correct (the CDC link is up to date the CRAN link is canonical), so let's revert it.

Reply:
Ah, these were edited directly in the README.md rather than in the README.Rmd